src/components/layers-panel/delete-confirmation.tsx

```diff
@@ -1,6 +1,9 @@
 import { ReactEventHandler, ReactNode } from "react";
 import styled, { DefaultTheme, useTheme } from "styled-components";
-import { color_accent_primary } from "../../constants/colors";
+import {
+  color_accent_primary,
+  color_accent_secondary,
```

Comment:
The color is different on dark and light theme. You need to add some color into theme.colors.